src/dist/replication/test/replica_test/unit_test/replica_test.cpp

```diff
@@ -59,5 +65,22 @@ TEST_F(replica_test, write_size_limited)
     ASSERT_EQ(get_write_size_exceed_threshold_count(), count);
 }
 
+TEST_F(replica_test, backup_request_qps)
+{
+    // create backup request
+    auto backup_request = dsn::message_ex::create_request(task_code());
```

Comment:
```suggestion
    message_ptr backup_request = dsn::message_ex::create_request(task_code());
```

You don't need `cleanup`.